firstvoices/backend/apps.py

```diff
@@ -4,7 +4,3 @@
 class BackendConfig(AppConfig):
     default_auto_field = "django.db.models.BigAutoField"
     name = "backend"
-
-    def ready(self):
-        import backend.models.signals  # noqa F401
-        import backend.search.signals  # noqa F401
```

Comment:
Is this intentional? This seems like it would disable all signals

Reply:
Ah the backend.models.signals was intentional but the search signals should still be there, I will re-add the line, thanks!